models/check-exist-item.js

```diff
@@ -1,15 +1,16 @@
 const db = require("../db/connection");
-exports.checkArticleExist = (article_id) => {
+
+exports.checkArticleExist = (article_id, table_name, column) => {
     return db
         .query(
-            `SELECT * FROM articles
-             WHERE article_id = $1;`,
+            `SELECT * FROM ${table_name}
+             WHERE ${column} = $1;`,
             [article_id]
         )
         .then(({ rows }) => {
             if (!rows.length) {
                 return Promise.reject({ status: 404, msg: "Not found!" });
             }
-            return rows;
+            // return rows;
         });
```

Comment:
article_id is also a foreign key so again, a separate check that the article exists should not be necessary as a psql error will be thrown.